### PR TITLE
Loading and Saving PyTorch Model; Use of logging

### PR DESCRIPTION
**Pull request by minseok-ryu**

### CODE REVIEW

**Comment by minseok-ryu** [comment]
After training, the trained model is saved in ".pt" in the "results" directory.
Regarding the "checkpoint", we can do that, but need more discussion what to save and how to save.
(web: https://pytorch.org/tutorials/beginner/saving_loading_models.html)

**Review by kibaekkim — CHANGES_REQUESTED** [review]
Please update the documentation.

File: src/appfl/config/config.py [hunk 1]
```diff
@@ -22,8 +22,16 @@ class Config:
     test_data_batch_size: int = 64
     test_data_shuffle: bool = False
 
-    # Results directory
-    result_dir: str = "./results"
+    # Initial Model Parameters
+    is_init_point: bool = True
```
Comment (kibaekkim) [inline src/appfl/config/config.py:26]: The default value would better be `False`, and the values below be empty string.

**Review by kibaekkim — CHANGES_REQUESTED** [review]
As this PR modifies the logging part in `utils.py`, it may be good to address #51 here.

**Review by kibaekkim — APPROVED** [review]

**Pull request merged**